backend/main.py

```
@@ -155,14 +163,176 @@ async def root():
+            "status": db_health["status"],
+            "type": db_health["database_type"],
+            "connected": db_health["connected"],
+            "table_count": db_health["table_count"],
+            "last_error": db_health.get("last_error")
+        }
+        
+        if db_health["status"] != "healthy":
+            health_status["status"] = "degraded"
+            
+    except Exception as e:
+        logger.error(f"Database health check failed: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)